GitFyle.Core.Api/Brokers/Storages/StorageBroker.cs

```diff
@@ -42,6 +42,14 @@ private async ValueTask<T> InsertAsync<T>(T entity)
             return entity;
         }
 
+        private async ValueTask<T> UpdateAsync<T>(T @object)
+        {
+            this.Entry(@object).State = EntityState.Modified;
+            await this.SaveChangesAsync();
```

Comment:
Remember to detach the object as we don't want the context to keep track of changes to an entity after it has been saved.

```suggestion
            await this.SaveChangesAsync();
            DetachSavedEntity(@object);
```